Automattic/wp-calypso
# Add sympathy dev warning

#### Proposed Changes
Adds a sympathy dev warning so developers are warned of the random local storage wipe out.

### Attempt 3 - Same as below but without Redux and using local storage.
### Attempt 2 - Adding to Initial redux state of notices
<img width="1727" alt="image" src="https://user-images.githubusercontent.com/3422709/199426604-a084ec02-0a1c-424e-aac9-135cecb76894.png">

### Attempt 1 - Direct DOM Attachment and removal 7fdb1c2263
<img width="1703" alt="image" src="https://user-images.githubusercontent.com/3422709/199454500-b9a31f55-1f29-4795-a6a1-861fe646fa2a.png">

#### Testing Instructions
- Keep refreshing a few times (At least 4 times) window and make sure the above notice appears. (Try  logged in and logged out(signup flow))


#### Pre-merge Checklist






Related to #
